spec/services/profiles/update_spec.rb

```diff
@@ -101,13 +101,13 @@ def sidekiq_assert_resave_article_worker(user, &block)
 
     it "enqueues resave articles job when changing bg_color_hex" do
       sidekiq_assert_resave_article_worker(user) do
-        described_class.call(user, profile: { brand_color1: "#12345F" })
+        described_class.call(user, user_settings: { brand_color1: "#12345F" })
```

Comment:
I needed to update the spec the reflect the new reality of where these attributes are stored.